odoo/models.py

```diff
@@ -2850,6 +2865,9 @@ def _setup_base(self):
         if cls._setup_done:
             return
 
+        # the classes that define this model's base fields and methods
+        cls._model_classes = tuple(c for c in cls.mro() if getattr(c, 'pool', None) is None)
```

Comment:
I think there's only 3-4 people in the entire R&D who know that `pool is None` means you're looking for definition classes, if someone other than us ever sees this, they're gonna have a bad time 🤣 

What I mean to say is: we should eventually try to use something else, preferably that exists in both definition and registry classes, even if it's just an alias for `getattr(c, 'pool', None) is None`

Reply:
Wait, I actually introduced helper functions for this...

Reply:
Oh damn, this code shouldn't be there.  It's a leftover from a rebase...